c/shared/CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ add_library(shared OBJECT
 set_property(TARGET shared PROPERTY CXX_STANDARD 17)
 
 target_include_directories(shared PRIVATE include resource)
+target_link_libraries(shared PUBLIC LibXml2::LibXml2)
```

Comment:
About this thing, `c/shared/include/ctlshare.h` has an `#include <libxml/tree.h>` so all consumers of `shared` includes must propagate the `libxml` includes, thus the `PUBLIC` linkage here. The issue is that the consumers of the includes do not consume `shared`, instead they consume the includes via `target_include_directories(rotatefont PRIVATE ../shared/include)` which bypasses the modern CMake designs. Could you look into fixing these issues?